### PR TITLE
Create a 0.29 release that works with Dart 2

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+#### 0.29.0+1 - 2018-07-19
```
Comment (nshahan) [inline CHANGELOG.md]: This version has already been published.
https://pub.dartlang.org/packages/quiver/versions/0.29.0+1

Comment (kevmoo) [reply CHANGELOG.md]: ack. Fixed.

**Comment by cbracken** [comment]
This was explicitly marked not LGTM preciously. The correct fix is to release new versions of built_*, not land a hack in quiver that violates semver.

**Comment by kevmoo** [comment]
> This was explicitly marked not LGTM preciously. The correct fix is to release new versions of built_*, not land a hack in quiver that violates semver.

This is a new PR. I dropped #471 and went w/ under the assumption that you were okay w/ this approach

https://github.com/google/quiver-dart/pull/471#issuecomment-406139459

**Pull request merged**